core/templates/components/state-directives/rule-editor/rule-editor.directive.html

```diff
@@ -1,38 +1,35 @@
-<div style="position: relative;">
+<div class="position-relative">
   <form class="form-inline protractor-test-rule-details" role="form" name="$ctrl.ruleEditForm.form">
     <div ng-if="$ctrl.rule.type != 'FuzzyMatches'" class="rule-parent">
       <rule-type-selector class="protractor-test-answer-description" local-value="<[$ctrl.rule.type]>" on-selection-change="$ctrl.onSelectNewRuleType">
       </rule-type-selector>
 
       <span ng-repeat="item in $ctrl.ruleDescriptionFragments track by $index" class="form-group oppia-rule-editor-span protractor-test-answer-description-fragment">
-        <span ng-if="item.type == 'select'" style="color: black;">
+        <span ng-if="item.type == 'select'" class="text-dark">
           <span ng-if="$ctrl.ruleDescriptionChoices.length === 1">
             "<[$ctrl.ruleDescriptionChoices[0].val|convertToPlainText|truncate]>"
           </span>
           <span ng-if="$ctrl.ruleDescriptionChoices.length !== 1">
-            <html-select style="width: 250px;" options="$ctrl.ruleDescriptionChoices" selection="$ctrl.rule.inputs[item.varName]">
+            <html-select class="oppia-rule-description-choice" options="$ctrl.ruleDescriptionChoices" selection="$ctrl.rule.inputs[item.varName]">
             </html-select>
           </span>
         </span>
 
-        <div style="max-height: 320px;
-        overflow-y: auto;
-        overflow-x: hidden;
-        text-overflow: ellipsis;">
-          <span ng-if="item.type == 'checkboxes'" style="color: black;">
+        <div class="oppia-items-container">
```

Comment:
This seems too vague. Pick a more specific name please -- oppia-custom-editor-container perhaps.